Public/Src/Engine/Processes/PipEnvironment.cs

```diff
@@ -49,14 +50,27 @@ public PipEnvironment()
         {
             FullEnvironmentVariables = GetFactory(ReportDuplicateVariable).PopulateFromEnvironment();
 
+#if PLATFORM_WIN
             var comspec = Path.Combine(SpecialFolderUtilities.SystemDirectory, "cmd.exe");
+            var pathExt = ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC";
+#endif
+
             var path =
                 string.Join(
-                    ";",
+                    OperatingSystemHelper.IsUnixOS ? ":" : ";",
                     SpecialFolderUtilities.SystemDirectory,
+#if PLATFORM_WIN
                     SpecialFolderUtilities.GetFolderPath(Environment.SpecialFolder.Windows),
-                    Path.Combine(SpecialFolderUtilities.SystemDirectory, "wbem"));
-            var pathExt = ".COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC";
+                    Path.Combine(SpecialFolderUtilities.SystemDirectory, "wbem")
+#else
+                    MacPaths.UsrBin,
+                    MacPaths.UsrSbin,
+                    MacPaths.Bin,
+                    MacPaths.Sbin

```

Comment:
This is a behavioral change to how we run pips. Were people manually adding them to PATH previously? Is there any concern that in some circumstances you might not want these included?

Reply:
When starting a new shell those PATH components are set by default. It's were the basic OS tools life. In a lot of scenarios we whitelist those paths when running sandboxed, this thus makes sure that we have a sane environment where basic tools are available for a pip to execute.